widget/embedded/src/types/config.ts

```diff
@@ -8,22 +8,16 @@ import type { Asset } from 'rango-sdk';
  * @property {string} background
  * @property {string} foreground
  * @property {string} primary
- * @property {string} success
- * @property {string} error
- * @property {string} warning
- * @property {string} surface
- * @property {string} neutral
+ * @property {string} secondary
  *
  */
+export type Colors = keyof WidgetColors;
 export type WidgetColors = {
```

Comment:
I was thinking of using a more meaningful name for the first variable. something like `WidgetColorsKeys`. What do you think?

Reply:
yes, it's better . fixed.